docs/src/assets/risk.csv

```diff
@@ -0,0 +1,1001 @@
+id,Risk

```

Comment:
what is the line ending you use in this file?

Reply:
Line ending?

Reply:
You have created german.csv with CRLF line endings. The rest of the files in the package uses LF line endings. I have checked myself. This file also uses CRLF endings.

It is OK to leave this this way. We just need to remember that we have such a situation.

If you do not know much about CRLF vs LF issue here is a good read: https://www.aleksandrhovhannisyan.com/blog/crlf-vs-lf-normalizing-line-endings-in-git/